docs/features/exception-handling.md

```diff
@@ -8,12 +8,39 @@ app.Features.AddExceptionHandling(...);
 
 ## Problem Details
 
-Adds an exception handler implementation that returns errors in problem details
-format. It also adds a middleware that logs exceptions.
+This feature implementation adds `.Net` `ProblemDetails` services with a custom
+`IExceptionHandler` implementation that returns errors in problem details
+format. It also adds a custom middleware that logs exceptions.
 
 ```csharp
 c => c.ProblemDetails(typeUrlFormat: "https://my-service.com/errors/{0}")
 ```
+`IExceptionHandler` implementation loops through `Baked.IExceptionHandler` 
+services and uses first handler which can handle current exception.

```

Comment:
I think there is a typo in here, you probably meant `ExceptionHandler` loops through the interfaces.

But this implementation detail is irrelevant anyway. To say "It loops all `IExceptionHandler` services and uses the first handler..." is sufficient.